wwwapp/settings.py

```diff
@@ -72,8 +72,11 @@
     'compressor',
     'wwwapp',
     'django_cleanup',
+    'social_django',
```

Comment:
Add a commentmentioning deprecation to allaccess. Also why at the end instead of next to the other one?